database/schemas/RelatedAlbumSchema.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const RelatedAlbumSchema = new Schema({
+  id: { type: String, required: true },
```

Comment:
if you have this author-permlink, so name it appropriately, otherwise you yourself will later get confused in the variables